src/client/datascience/data-viewing/types.ts

```diff
@@ -96,8 +96,7 @@ export enum ColumnType {
     Bool = 'bool'
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export type IRowsResponse = any[];
+export type IRowsResponse = Record<string, unknown>[];
```

Comment:
In our code, we know exactly what we're expecting (or what its supposed to look like), hence provided better typing

Reply:
In the WebView we're expecting an array of objects